libs/ledgerjs/packages/hw-app-eth/src/modules/EIP712/EIP712.utils.ts

```diff
@@ -321,13 +322,24 @@ export const getSchemaHashForMessage = (message: EIP712Message): string => {
  * @param {EIP712Message} message
  * @returns {MessageFilters | undefined}
  */
-export const getFiltersForMessage = (
-  message: EIP712Message
-): MessageFilters | undefined => {
+export const getFiltersForMessage = async (
+  message: EIP712Message,
+  remoteCryptoAssetsListURI?: string | null
+): Promise<MessageFilters | undefined> => {
   const schemaHash = getSchemaHashForMessage(message);
   const messageId = `${message.domain?.chainId ?? 0}:${
     message.domain?.verifyingContract ?? NULL_ADDRESS
   }:${schemaHash}`;
 
-  return EIP712CAL[messageId];
+  try {
+    if (remoteCryptoAssetsListURI) {
+      const { data: dynamicCAL } = await axios.get(
+        `${remoteCryptoAssetsListURI}/eip712.json`
+      );
+      return dynamicCAL[messageId] || EIP712CAL[messageId];
```

Comment:
is `EIP712CAL` generated ? Seems like the file is empty

Reply:
Yes it's a generated file. But it's a different process, and it shouldn't be generated in this PR.